visualization/app/codeCharta/ui/artificialIntelligence/artificialIntelligence.component.ts

```diff
@@ -40,34 +40,37 @@ interface MetricSuggestionParameters {
 	isOutlier?: boolean
 }
 
+interface RiskProfile {
+	lowRisk: number
+	moderateRisk: number
+	highRisk: number
+	veryHighRisk: number
+	totalRloc?: number
+}
+
+const HEIGHT_METRIC = "mcc"
+const AREA_METRIC = "rloc"
+const EXCLUDED_FILES = new Set(["html", "sass", "scss", "scss", ".txt", "md", "json", undefined])
```

Comment:
```suggestion
const EXCLUDED_FILES = new Set(["html", "sass", "scss", ".txt", "md", "json", undefined])
```

Reply:
Also I'd rename this to `EXCLUDED_FILE_EXTENSIONS`

Reply:
We don't need the point in `.fileExtension` because it's sliced in getFileExtension(fileName: string) method and I don't the any point to change this method.

Reply:
I'm confused. :D 
a few things here, I'd change:
- Rename to `EXCLUDED_FILE_EXTENSIONS` since this is a set of file extensions. It's more expressive than just calling it `EXCLUDED_FILES`. A string is not a file.
- The element `scss` is listed twice. It can be removed once.
- The dot in `.txt` can be removed I guess.

Reply:
You're right. I was a bit sloppy.